client/python/gradio_client/utils.py

```diff
@@ -38,6 +42,55 @@ class InvalidAPIEndpointError(Exception):
     pass
 
 
+class Status(Enum):
+    STARTING = "STARTING"
+    JOINING_QUEUE = "JOINING_QUEUE"
+    QUEUE_FULL = "QUEUE_FULL"
+    IN_QUEUE = "IN_QUEUE"
+    SENDING_DATA = "SENDING_DATA"
+    START_ITERATION = "START_ITERATION"
```

Comment:
Using the terms iteration and iterating to be consistent with generator functions. But maybe we can rename `START_ITERATION`  to `PROCESSING` instead?

Reply:
I prefer PROCESSING as well